src/style.scss

```diff
@@ -0,0 +1,111 @@
+@import "variables";
+@import "normalize";
+
+.product-card {
+  width: 200px;
+  margin: auto;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  cursor: pointer;
+
+  &__photo {
+    margin-top: 16px;
+    margin-bottom: 36px;
+    text-align: center;
+
+    .product-card__img {
+      height: 134px;
+    }
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColor;
+    text-decoration: none;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondaryColor;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
```

Comment:
generally, not a mistake in this separate case, but `stars` might be a separate BEM-block (so styles should be in a separate file in this case)